ui/src/components/SignInDialog/index.jsx

```diff
@@ -117,7 +118,10 @@ export default class SignInDialog extends Component {
                 <ListItemText primary="Sign in with GitHub" />
               </ListItem>
             )}
-            <ListItem button onClick={this.handleCredentialsDialogOpen}>
+            <ListItem
+              disabled={thirdPartyLogin}
```

Comment:
> I don't think they'd ever see the sign-in menu while at /third-party. Maybe I'm missing something?

The sign-in menu is simply a modal and can be displayed on any view. You can be at `/third-party` and still see the sign-in menu.

> Also, it may make sense to conditionalize this on just having /third-party as the current URL path

👍